From a570a67b8a16c98cca8221b1748e2e5586ebd841 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 27 Jul 2020 05:01:25 -0700
Subject: [PATCH 1467/1852] dt-bindings: media: vpss-csc: Fix description of
 color conversion

commit 50609a6c8cc505850eaae210bac326c61fb8b4b0 from
https://github.com/Xilinx/linux-xlnx.git

Fix the color conversion description as RGB or YUV(444, 422, 420) to
RGB or YUV(444, 422, 420) instead of RGB to YUV only.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt       | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
index b3627af85e6e..d2bad87442ee 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-csc.txt
@@ -1,7 +1,8 @@
 Xilinx VPSS Color Space Converter (CSC)
 -----------------------------------------
 The Xilinx VPSS Color Space Converter (CSC) is a Video IP that supports
-color space conversion from RGB input to YUV output.
+color space conversion from RGB or YUV(444, 422, 420)input to RGB or
+YUV(444, 422, 420) output.
 
 Required properties:
 
-- 
2.31.1

